pandas/tests/groupby/test_nth.py

```diff
@@ -94,6 +94,16 @@ def test_nth_with_na_object(index, nulls_fixture):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("method", ["first", "last"])
+def test_first_last_with_None(method):
+    # https://github.com/pandas-dev/pandas/issues/32800
```

Comment:
can you add a comment, that we wish to preserve None in object dtypes.

Reply:
Done